dom-event-delegation/main.js

```diff
@@ -0,0 +1,14 @@
+const $taskList = document.querySelector('.task-list');
+
+function taskList(event) {
+  console.log('event.target: ', event.target);
+  console.log('element tag name: ', event.target.tagName);
+
+  if (event.target.tagName === 'button') {
```

Comment:
This is incorrect. When you click on the Done button, are your tasks getting removed? If not, something is wrong with your code then right?